<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QubesTemplateManager</class>
 <widget class="QMainWindow" name="QubesTemplateManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qubes Template Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="listing">
      <property name="editTriggers">
       <set>QAbstractItemView::AllEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="infobox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Click the 'Status' column to change the status of templates.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionInstall"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionInstall">
   <property name="icon">
    <iconset theme="go-down">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Install</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
